content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Using AWS Remotes in DVC
```

Comment:
In the title, description, and intro, should we assume users are familiar with the concept of "remotes"? Would it better to call it something like "Syncing Data to AWS S3"?

Reply:
I like that! It's probably better for SEO too.